src/wweb/middleware/middleware.ts

```diff
@@ -20,6 +20,8 @@ export const middleware = (
   return async (msg: Message) => {
     // if onlyOwner is true, the message will only be processed if it is sent by the owner,
     // otherwise it will only be processed if it is not sent by the owner
+    if (process.env.NODE_ENV === 'development') return next(msg);
```

Comment:
move this to here: https://github.com/vinibgoulart/wpp-playground/pull/31/files#diff-70404c349283f04f411c8c76bf6d1eb75523979d0be19c18fc539179a90ec39aR29
![image](https://github.com/vinibgoulart/wpp-playground/assets/88122830/d0c15341-cd59-4d5a-a370-110824f4ee4e)

only for message owner middleware

Reply:
```tsx
} else {
	const isDevelopment = process.env.NODE_ENV === 'development';
      if (msg.fromMe && !isDevelopment) {
        return;
      }
    }
```

Reply:
avoid else

Reply:
Why?